source/includes/language-compatibility-table-node.rst

```diff
@@ -226,3 +240,5 @@
      - ✓
      - ✓
 
+.. [#latest-note] Version 6.0.x of the {+driver-short+} requires the latest version
+   of Node.js v16 or later.
```

Comment:
Q: Do we know what the minimum version of 16 is? The way this reads seems like every time a new release of Node.js comes out then that will now be the only version that works with our driver, is that true? Or is it compatible with whatever is currently the latest version + any future releases?